app/views/docs/index.phtml

```diff
@@ -51,7 +51,17 @@ $cols = [
                 </ul>
 
                 <div class="margin-bottom-small">
-                    <b class="text-size-small">REST API</b>
+                    <b class="text-size-small">APIs</b>
+                </div>
+
+                <ul class="margin-bottom">
+                    <li><a href="/docs/rest">REST</a></li>
+                    <li><a href="/docs/graphql">GraphQL</a></li>
+                    <li><a href="/docs/realtime">Realtime</a></li>
+                </ul>
+
+                <div class="margin-bottom-small">
+                    <b class="text-size-small">References</b>
```

Comment:
Would it make more sense to call this Services rather than References?

Reply:
I agree that might fit better with our terminology. @eldadfux What do you think?

Reply:
I fear it will create confusion with `Services` vs `Guides` since you'll see both link labels in both categories. Don't really see a downside in `References` as this is what the content of this category includes.

Reply:
What if we called them `API References`, what I think is confusing is that people are going to look for the keyword `API` and not see it.

🤔 Would it be too long?